charts/graphistry-helm/templates/caddy/caddy-deployment.yaml

```diff
@@ -26,11 +21,6 @@ spec:
 {{- end }}
   template:
     metadata:
-      annotations:
-        autoheal: "true"
```

Comment:
out of curiosity: why we don't need anymore the autoheal annotations?

Reply:
good catch i missed a couple